jrcc-access-spring-boot-autoconfigure/src/test/java/ca/bc/gov/open/jrccaccess/autoconfigure/AccessAutoConfigurationTests.java

```diff
@@ -35,4 +36,13 @@ public void constructor_with_null_output_property_should_throw_OutputConfigMissi
         properties.setOutput(null);
         aac = new AccessAutoConfiguration(properties);
```

Comment:
we go with `sut` for SystemUnderTest,  what aac stands for?

Reply:
aac = access auto configuration

Reply:
I just realized after putting my comment, damn

Reply:
I was like, Assert Act, ... something, hmmm what is it

Reply:
I will put a not in the readme I guess that `sut` = system under test

Reply:
because that's kind the only abbreviation that is allowed, ...

Reply:
Haha all good, so would you like me to change `aac` to `sut`? The abbreviation `aac` is used elsewhere in the file too

Reply:
yes